prediction/markets.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = [
+  {
+    questionId:
+      "0x0000000000000000000000000000000000000000000000000000000000000001",
```

Comment:
What's this do? Do we only need 1 question ID for now?

Reply:
Read up from the gnosis conditional tokens docs, I guess for our usecase the question is always "who will complete the objective first?" and then we have the objective live separately? not really sure how to best structure that, any thoughts?

Reply:
as we add new markets we can add here. we also need to supply a max number of outcomes at the start too.

Reply:
right now i maintain a map from question id to title as a json in the ui directory:
ui/const/betting/config.sepolia.json